packages/paste-core/components/menu/src/SubMenuButton.tsx

```diff
@@ -1,24 +1,31 @@
 import * as React from 'react';
+import type {BoxElementProps} from '@twilio-paste/box';
 import type {MenuPrimitiveButtonProps} from '@twilio-paste/menu-primitive';
 import {MenuPrimitiveButton} from '@twilio-paste/menu-primitive';
 import {MediaObject, MediaBody, MediaFigure} from '@twilio-paste/media-object';
 import {ChevronRightIcon} from '@twilio-paste/icons/esm/ChevronRightIcon';
 import {StyledMenuItem} from './MenuItem';
 
-export type SubMenuButtonProps = MenuPrimitiveButtonProps;
+export type SubMenuButtonProps = MenuPrimitiveButtonProps & {element?: BoxElementProps['element']};
 
-const SubMenuButton = React.forwardRef<HTMLButtonElement, SubMenuButtonProps>((props, ref) => {
-  // MenuPrimitiveButton from reakit types `as` as HTML element names, but accepts components. any prevents type errors
-  return (
-    <MenuPrimitiveButton {...props} as={StyledMenuItem as any} ref={ref}>
-      <MediaObject as="span" verticalAlign="center">
-        {props.children && <MediaBody as="span">{props.children}</MediaBody>}
-        <MediaFigure as="span" align="end" spacing="space20">
-          <ChevronRightIcon decorative size="sizeIcon30" />
-        </MediaFigure>
-      </MediaObject>
-    </MenuPrimitiveButton>
-  );
-});
+const SubMenuButton = React.forwardRef<HTMLButtonElement, SubMenuButtonProps>(
+  ({element = 'SUBMENU', ...props}, ref) => {
+    // MenuPrimitiveButton from reakit types `as` as HTML element names, but accepts components. any prevents type errors
+    return (
+      <MenuPrimitiveButton {...props} as={StyledMenuItem as any} element={`${element}_BUTTON`} ref={ref}>
+        <MediaObject as="span" verticalAlign="center" element={`${element}_MEDIA_OBJECT`}>
+          {props.children && (
+            <MediaBody as="span" element={`${element}_MEDIA_BODY`}>
```

Comment:
This _should_ be SUBMENU_BUTTON_MEDIA_BODY, because it's the media body in the SubMenuButton component, not the actual SUBMENU that appears as a result of clicking this.